pkg/controllers/provisioningv2/rke2/bootstrap/controller_test.go

```diff
@@ -4,11 +4,15 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 	"fmt"
+
+	provisioningcontrollers "github.com/rancher/rancher/pkg/generated/controllers/provisioning.cattle.io/v1"
+	"github.com/rancher/rancher/pkg/provisioningv2/rke2/installer"
 	"testing"
 
 	"github.com/rancher/rancher/pkg/controllers/provisioningv2/rke2"
 	"github.com/rancher/rancher/pkg/settings"
 
+	provcontrollers "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
```

Comment:
```suggestion
	provv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
```
My bad, should've clarified this.